website/docs/r/policy_project.html.markdown

```diff
@@ -47,6 +45,7 @@ In addition to arguments listed above, the following attributes are exported:
 * `id` - ID of the resource.
 * `revision` - Indicates current revision number of the object as seen by NSX-T API server. This attribute can be useful for debugging.
 * `path` - The NSX path of the policy resource.
+* `_default` - The server will populate this field when returing the resource. Ignored on PUT and POST.true - the project is a default project. Default projects are non-editable, system create ones.
```

Comment:
I don't think it is present in the resource at all? I'm not sure its useful, but if you'd like to add it, I think it should be named `default` or `is_default` to follow convention (we have `is_default` in tz and security policy data sources)

Reply:
I think that it's useless. Maybe it's better if I'll remove it...

Reply:
But maybe in a separate PR as this one is regarding docs - I'll get rid of this in the resource as well.

Reply:
I don't see it in resource schema: https://github.com/vmware/terraform-provider-nsxt/blob/master/nsxt/resource_nsxt_policy_project.go#L27, am I missing something?

Reply:
Cool so I removed it already :)
I'll just fix the doc then. Thanks!!